domain/timeline/src/commonMain/kotlin/social/androiddev/domain/timeline/HomeTimelineRepository.kt

```diff
@@ -0,0 +1,13 @@
+package social.androiddev.domain.timeline
+
+import kotlinx.coroutines.flow.Flow
+
+import org.mobilenativefoundation.store.store5.StoreResponse
+import social.androiddev.domain.timeline.model.StatusUI
+
+interface HomeTimelineRepository {
+    suspend fun read(feedType: FeedType, refresh: Boolean = false): Flow<StoreResponse<List<StatusUI>>>
+}
+sealed class FeedType(val type:String){
+    object Home: FeedType("HOME")
```

Comment:
I think the domain model `FeedType` could just be an Enum. And you can map this Enum -> JSON string that needs to be passed to the API in the RealHomeTimelineRepostory. 

Just a thought, I don't feel strongly.